web/source/kmwosk.js

```diff
@@ -891,6 +971,12 @@ if(!window['tavultesoft']['keymanweb']['initialized']) {
             case 'K_SPACE':
               keymanweb.KO(0, Lelem, ' ');
               break;
+            case 'K_CAPS':
+            case 'K_NUMLOCK':
+            case 'K_SCROLL':
```

Comment:
Do we need to consider `K_SCROLLLOCK` as well?

Reply:
I don't see a definition for K_SCROLLLOCK in the `osk.keyCodes` array.  Should it exist?  I looked for any variations when writing this, but the lack of its definition caused me to not consider it.